### ISSUE TITLE
Bforartists version 3.3.2

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#3305]
With a release candidate of Blender 3.3.2 in the builder page we should start to prepare for a BFA release too.

I would say we wait until the Blender version goes live, which i expect to happen tomorrow, then make a new merge, tackle what comes in with this merge, and then do our 3.3.2 release.

Do we need a new splash?

**Comment by Draise14** [comment]
Since it's a point update, I guess we can stick to the same. Can I use the official fonts I've been using? I'll commit the splash to a branch here. 

I would prefer 3D images made with Bforartists always, half the art community hates AI and NFTs, so best no drama.

**Comment by ReinerBforartists** [comment]
I love AI <3

But sure, you make it, you decide. But please this time update the splash template ... ;)

**Comment by Draise14** [comment]
The Inkscape template is up to date with the latest fonts from the File repository. I guess I'll have to update the gimp one too.

**Comment by ReinerBforartists** [comment]
Yes please. As told, the image is needed to be part of the repo in an editable way. And the gimp template suits this job :)

**Comment by ReinerBforartists** [comment]
Ah, wait, i already did for 3.3.1. All well. With then next splash then :)

**Comment by Draise14** [comment]
I'll have to install Gimp - last time I used it it took like 7 minutes to even load up. What's the version you typically use?

**Comment by ReinerBforartists** [comment]
First time load is a bit slow, it scans everything for fonts and plugins. Second startup should be a bit faster. I currently use Gimp 2.10.30. Newest is 2.10.32. The upcoming version 3 is upcoming since i think five years or so ...

**Comment by ReinerBforartists** [comment]
They are kidding me. The current Blender version fails at Windows to compile. Investigating ...

**Comment by ReinerBforartists** [comment]
Better wait with the next merge until it compiles again!

![notcompiling](https://user-images.githubusercontent.com/13919702/201513886-39105572-47b2-493e-a59a-9b357dd67ae2.jpg)
![notcompiling](https://user-images.githubusercontent.com/13919702/201513889-857a73af-797c-427e-bfe3-33ea06d32472.jpg)

**Comment by ReinerBforartists** [comment]
Compiles again.

May i at this point remind to do a merge from time to time? It becomes too much when you do it all two weeks. And even when next wednesday or thursday the next version may arrive, we need to go through the log and fix what has changed or broke by the merge. And the longer you wait the more it becomes.

**Comment by Draise14** [comment]
Yes sure! You can do a merge when you'd like. I've sponsored Iyad to get the merge this week but he was tied up with Uni.. and I agree, more merges the better. 

My plan next year is to only have one client at a time, so I can work Bforartists and the personal work without a problems. Then I too can do merges.

**Comment by ReinerBforartists** [comment]
I'll have a look. Today and tomorrow i have some work to do though.

**Comment by ReinerBforartists** [comment]
Ah, what the heck. Merging.

**Comment by ReinerBforartists** [comment]
Still no version 3.3.2. Seems that they are kidding us ^^

**Comment by iyadahmed** [comment]
No 3.3.2 blender? :o

**Comment by ReinerBforartists** [comment]
Not yet. RC is still in the builder page though ^^

**Comment by ReinerBforartists** [comment]
I think we can close this task. ETA for version 3.4 is in two weeks. December 7 ...

https://developer.blender.org/project/view/144/
